bin/migration/tables/users.py

```diff
@@ -0,0 +1,56 @@
+from .helpers import check_existing_record, parse_to_timestamp, audit_entry_creation, log_failed_imports
+
+class UserManager:
+    def __init__(self, source_cursor):
+        self.source_cursor = source_cursor
+        self.failed_imports = set()
+
+    def get_data(self):
+        self.source_cursor.execute("SELECT * from public.users")
+        return self.source_cursor.fetchall()
+
+    def migrate_data(self, destination_cursor, source_user_data):
+        batch_users_data = []
+
+        for user in source_user_data:
+            id = user[0]
+
+            if not check_existing_record(destination_cursor,'users', 'id', id):
+                first_name = user[12]
+                last_name = user[13]
+                email = user[6]
+                organisation = user[8]
+                phone = user[7]
+                created_at = parse_to_timestamp(user[15])
+                modified_at = parse_to_timestamp(user[17])
+                created_by = user[14]
```

Comment:
@mazzopardi2  Have we covered the existing data for 'email confirmed', loginenabled' and invited' fields at the migrated data?

Reply:
@JeyaSolirius We have a field in Portal Access which will record the details of the user's status, see portal_access table status field will take in 'invitation_sent', 'registered', 'active', 'inactive'

The deleted_at flag will be used to record any deleted users